src/navigation/appNavigation.js

```diff
@@ -202,6 +206,13 @@ const assetsFlow = createStackNavigator(
 
 assetsFlow.navigationOptions = hideTabNavigatorOnChildView;
 
+// SELF FLOW
```

Comment:
Shall we rename it to `ME FLOW`? Only because we started to use that term everywhere

Reply:
Updated